### PR TITLE
Update workshop-syllabus.md

### PR DESCRIPTION
**Pull request by mblue9**

### CODE REVIEW

File: dev/workshop-syllabus.md [hunk 1]
```diff
@@ -42,25 +42,22 @@ Sudents will be expected to participate in the workshop in a hands-on way, follo
...
 ## Learning goals
 
-* To understand of the recurrent concepts and steps of bulk RNA sequencing data analysis
-* To approach data representation and analysis though a tidy paradigm, integrating tidyverse with tidybulk.
+* To understand the key concepts and steps of bulk RNA sequencing data analysis
+* To approach data representation and analysis though a tidy data paradigm, integrating tidyverse with tidybulk.
 
 ## Learning objectives
 
-* To remember the recurrent concept and vocabulary of RNA sequencing data analysis
```
Comment (stemangiola) [inline dev/workshop-syllabus.md]: With this section they explicitly asked to follow their system , remember, understand, apply.. all points with key words. It was specified in the original document.

File: dev/workshop-syllabus.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-This can be achieved for RNA sequencing data with tidybulk, tidyHeatmap and tidyverse packages. The package tidybulk provides a modular framework and a tidy data structure for bulk transcriptional analyses. tidyHeatmap provides a tidy implementation of the function ComplexHeatmaps::Heatmap. These packages are part of the tidytranscriptomics suite that introduces semantics and properties of a tidy approach to RNA sequencing data.
+This can be achieved for RNA sequencing data with the tidybulk, tidyHeatmap and tidyverse packages. The package tidybulk provides a tidy data structure and a modular framework for bulk transcriptional analyses. tidyHeatmap provides a tidy implementation of ComplexHeatmap. These packages are part of the tidytranscriptomics suite that introduces a tidy approach to RNA sequencing data.
 
 The topics presented in this workshop will be
 
 - Data exploration
 - Data preprocessing and quality control
 - Data dimensionality reduction
-- Differential transcript abundance analyses 
+- Differential gene abundance analyses
```
Comment (stemangiola) [inline dev/workshop-syllabus.md]: I would prefer to say transcript, as gene abundance is technically incorrect. As gene is just a piece of information, gene abundance would be at most gene duplication on the genome.

File: dev/workshop-syllabus.md [hunk 3]
```diff
@@ -42,25 +42,22 @@ Sudents will be expected to participate in the workshop in a hands-on way, follo
 |----------------------------------------|------|
 | Data import and quality control        | 30m  |
 | Data preprocessing and visualisation   | 30m  |
-| Differential transcript abundance      | 30m  |
+| Differential gene abundance            | 30m  |
```
Comment (stemangiola) [inline dev/workshop-syllabus.md]: See above because I would like transcript rather than gene. :)

File: dev/workshop-syllabus.md (hunk 1 above)
Comment (mblue9) [reply dev/workshop-syllabus.md]: As far as I understand it those are just examples they have provided, they just want us to include some e.g. they say "Some examples:" in the original template. Here's an example of the plyranges workshop syllabus from last year: https://github.com/sa-lee/plyrangesWorkshops

File: dev/workshop-syllabus.md (hunk 2 above)
Comment (mblue9) [reply dev/workshop-syllabus.md]: What about using differential gene expression instead? As wondering if differential transcript analysis might get confused with differential isoform analysis? e.g.  from [here](https://liorpachter.wordpress.com/2019/01/07/fast-and-accurate-gene-differential-expression-by-testing-transcript-compatibility-counts/) "Differential transcript expression (DTE) refers to change in one of the isoforms"

**Comment by stemangiola** [comment]
That's a good example. Sure feel free then to make changes taking this aS
reference. I will be flying for the next 24 hours. Happy to go with what
you propose

**Comment by stemangiola** [comment]
For me the thing we are me surfing is differential abundance of something
(transcript). But I also realise that the scientific community settled on a
terminology, if I think that is wrong mtaters not much. If not:
differential (transcript) abundance analysis,  feel free to use what you
you think is best. :)

**Comment by stemangiola** [comment]
Sorry I wrote with phone with some random autocompletion. The bottom-line that I will not get too fussy about it 😉 Best wishes.Stefano  Stefano Mangiola | Postdoctoral fellowPapenfuss LaboratoryThe Walter Eliza Hall Institute of Medical Research+61 (0)466452544 From: stefanoSent: Saturday, 29 February 2020 9:29 PMTo: stemangiola/tidybulkCc: stemangiola/tidybulk; CommentSubject: Re: [stemangiola/tidybulk] Update workshop-syllabus.md (#30) For me the thing we are me surfing is differential abundance of something (transcript). But I also realise that the scientific community settled on a terminology, if I think that is wrong mtaters not much. If not: differential (transcript) abundance analysis,  feel free to use what you you think is best. :) On Sat, 29 Feb 2020, 1:20 PM mblue9 <notifications@github.com> wrote:@mblue9 commented on this pull request.In dev/workshop-syllabus.md:>   The topics presented in this workshop will be  - Data exploration - Data preprocessing and quality control - Data dimensionality reduction-- Differential transcript abundance analyses +- Differential gene abundance analyses What about using differential gene expression instead? As wondering if differential transcript analysis might get confused with differential isoform analysis? e.g. from here "Differential transcript expression (DTE) refers to change in one of the isoforms"—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by mblue9** [comment]
Ok have made some more edits, see what you think. I went with differential gene expression analysis for the moment :)

**Pull request merged**